### DISCUSSION TITLE
How do I make an helicopter crash when using the debugger?

### DISCUSSION THREAD

**Discussion by ASHBLY33** [TEHE-Studios/ExpandedHelicopterEvents#261] [Q&A] [answered]
I tried to make an Military cargo chopper trough the debugger but couldn't, are all helicopters scripted to crash and how do I do it trough the debug mode?

**Comment by Chuckleberry-Finn** [comment]
Some cannot crash at all.

**Reply by ASHBLY33** [reply]
Wich ones?, and how do I know where they landed? also, when spawning trough debug menu, are they still afected by the weather?

**Reply by Chuckleberry-Finn** [reply, marked as answer]
It is not marked or noted anywhere.
There is a marker where they crashed on screen for some time.
Yes, they're still impacted by weather but there are sandbox options now to toggle this.
